example_configs/thelounge.md

```diff
@@ -2,15 +2,41 @@
 
 [Main documentation here.](https://thelounge.chat/docs/configuration#ldap-support)
 
-For simple user auth with LLDAP on localhost adapt this in the main config.js:
+Simple Config:
 
 ```
-ldap: {
-  enable: true,
-  url: "ldap://127.0.0.1:3890",
-  tlsOptions: {},
-  primaryKey: "uid",
-  baseDN : "ou=people,dc=example,dc=com",
+      ldap: {
+        enable: true,
+        url: "ldap://localhost:389",
+        tlsOptions: {},
+        primaryKey: "uid",
+        baseDN: "ou=people,dc=example,dc=com",
+    },
 ```
 
-And comment out with ```//``` the entire ```searchDN``` block.
+In this config, The Lounge will use the credentials provided in web ui to authenticate with lldap. It'll allow access if authentication was successful and the user is a member of Base DN.
+
+
+Advanced Config:
+
+```
+      ldap: {
+        enable: true,
+        url: "ldap://localhost:389",
+        tlsOptions: {},
+        primaryKey: "uid",
+        searchDN: {
+            rootDN: "uid=ldap-viewer,ou=people,dc=example,dc=com",
+            rootPassword: ""
+            filter: "(memberOf=cn=thelounge,ou=groups,dc=example,dc=com)",
+            base: "dc=example,dc=com",
+            scope: "sub",
+        },
+    },
+```
+
+`rootDN` is similar to bind DN in other applications. It is used in combination with `rootPassword` to query lldap. `ldap-viewer` user in `lldap` is a member of the group `lldap_strict_readonly` group. This gives `ldap-viewer` user permission to query `lldap`.
```

Comment:
Can you reset your password via the lounge? Also, the readonly permission is redundant, password manager is a superset.

Reply:
@nitnelave Yep, Updated this in the next commit